### PR TITLE
Orange peel PR

### PR DESCRIPTION
**Pull request by jdevenny**
Make sure the peel is included in the Tangerine.

### CODE REVIEW

**Comment by jdevenny** [comment]
@sherifgmansour - what's the deal?

**Comment by jdevenny** [comment]
#2

File: peel.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+orange peel!
+
```
Comment (jdevenny) [inline peel.txt:2]: test

Comment (jdevenny) [reply peel.txt:2]: hello

File: peel.txt [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+
+
```
Comment (jdevenny) [inline peel.txt:4]: why the whitespace?

Comment (jdevenny) [reply peel.txt:4]: TABS OVER SPACES

**Comment by kannonboy** [comment]
Test comment

**Review by sherifgmansour — APPROVED** [review]
LGTM

**Review by kannonboy — COMMENTED** [review]
LGTM

**Pull request merged**